pkg/antctl/raw/check/cluster/command.go

```diff
@@ -34,13 +34,15 @@ import (
 )
 
 func Command() *cobra.Command {
+	o := newOptions()
 	command := &cobra.Command{
 		Use:   "cluster",
 		Short: "Runs pre installation checks",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return Run()
+			return Run(o)
 		},
 	}
+	command.Flags().StringVar(&o.testImage, "test-image", o.testImage, "Container image override for the cluster checker")
```

Comment:
Will you update the usage doc for this change?

Reply:
We didn't document other options like "--run", and cover them via https://github.com/antrea-io/antrea/blob/a03c850e28cf042a5bdafe99670502cbdbc2ac30/docs/antctl.md?plain=1#L148-L152

I think that doc just gives basic examples and we don't want to maintain every option there.